content/en/continuous_integration/setup_tests/dotnet.md

```diff
@@ -13,55 +13,55 @@ further_reading:
 ## Compatibility
 
 Supported .NET versions:
-* .NET Core 2.1+
-* .NET Core 3.0+
-* .NET 5.0+
+* .NET Core >= 2.1 and >= 3.0
+* .NET >= 5.0
 
 Supported test frameworks:
-* xUnit 2.2+
-* NUnit 3.0+
-* MsTest V2 14+ (experimental)
-
-Supported CI providers:
-* Appveyor
-* Azure Pipelines
-* BitBucket
-* BuildKite
-* CircleCI
-* GitHub Actions
-* GitLab
-* Jenkins
-* TravisCI
+* xUnit >= 2.2
+* NUnit >= 3.0
+* MsTest V2 >= 14
 
 ## Prerequisites
 
 [Install the Datadog Agent to collect tests data][1].
 
-## Installing tracing
+## Installing the .NET tracer
 
 To install the `dd-trace` command globally on the machine, run:
 
 {{< code-block lang="bash" >}}
 dotnet tool install -g dd-trace
 {{< /code-block >}}
 
-Or, if you have a previous version of the tool, update it by running:
+## Instrumenting tests
+
+### xUnit and NUnit
+
+To instrument your test suite, prefix your test command with `dd-trace`, providing the name of the service or library under test as the `--dd-service` parameter, and the environment where tests are being run (for example, `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) as the `--dd-env` parameter. For example:
 
 {{< code-block lang="bash" >}}
-dotnet tool update -g dd-trace
+dd-trace --dd-service=my-dotnet-app --dd-env=ci -- dotnet test
 {{< /code-block >}}
 
-## Instrumenting your tests
+All tests will be automatically instrumented.
+
+### MsTest V2
+
+To instrument your test suite, prefix your test command with `dd-trace`, providing the name of the service or library under test as the `--dd-service` parameter, and the environment where tests are being run (for example, `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) as the `--dd-env` parameter.
 
-To instrument your test suite, prefix your test command with `dd-trace`. For example:
+You will also need to configure the tracer to use Call Target instrumentation by setting `DD_TRACE_CALLTARGET_ENABLED=true`, which is not the default value.
+
+For example:
 
 {{< code-block lang="bash" >}}
-dd-trace dotnet test
+DD_TRACE_CALLTARGET_ENABLED=true dd-trace \
+  --dd-service=my-dotnet-app --dd-env=ci -- \
+  dotnet test
```

Comment:
This is a common thing in bash but not supported in the default Windows command line (unless customer is using WSL2). Due the majority of users of .NET uses Windows, maybe we should add a windows compatible version of this script. 
See: 
- https://superuser.com/questions/424001/launch-windows-program-with-custom-environment-variable
- https://stackoverflow.com/a/34769146

Reply:
For **CMD**:
```Batchfile
SET DD_TRACE_CALLTARGET_ENABLED=true && dd-trace --dd-service=my-dotnet-app --dd-env=ci -- dotnet test
```
For **PowerShell**:
```PowerShell
$env:DD_TRACE_CALLTARGET_ENABLED="true"; dd-trace --dd-service=my-dotnet-app --dd-env=ci -- dotnet test
```